docs/styleguide.md

```diff
@@ -184,7 +184,7 @@ of argument is notated by either the common types:
 
 If an argument or a method is unique to certain platforms, those platforms are
 denoted using a space-delimited italicized list following the datatype. Values
-can be `macOS`, `Windows`, or `Linux`.
+can be `macOS`, `Windows` or `Linux`.
```

Comment:
No oxford commas allowed, eh? 😛

Reply:
The docs parser doesn't understand them :laughing: